seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/CoordinatorService.java

```diff
@@ -141,6 +148,8 @@ public CoordinatorService(@NonNull NodeEngineImpl nodeEngine, @NonNull SeaTunnel
         this.engineConfig = engineConfig;
         masterActiveListener = Executors.newSingleThreadScheduledExecutor();
         masterActiveListener.scheduleAtFixedRate(this::checkNewActiveMaster, 0, 100, TimeUnit.MILLISECONDS);
+        monitorJob = Executors.newSingleThreadScheduledExecutor();
```

Comment:
Hi, we already have `monitorService` in SeaTunnelServer.java, please use it to printJobDetailInfo. Don't create new ScheduledExecutor. Judging that if it is the master node, just print the corresponding information in `printExecutionInfo`.